Scoop/Controller/Onboarding/OnboardingPageViewController.swift

```diff
@@ -93,6 +93,17 @@ extension OnboardingPageViewController: UIPageViewControllerDataSource {
         return index
     }
     
+    func setupNextButton(view: UIView, action: UIAction, button: UIButton) {
+        button.setImage(UIImage(named: "nextbutton"), for: .normal)
+        button.clipsToBounds = true
+        button.addAction(action, for: .touchUpInside) //EDIT LATER
+        view.addSubview(button)
```

Comment:
You can access the `view` for this ViewController anywhere in this VC, so you don't have to pass it in through a parameter. You can also set the action for this button inside this function too